src/components/Cluster/ClusterDetail/NodePool.tsx

```diff
@@ -270,16 +270,6 @@ class NodePool extends Component<INodePoolsProps, INodePoolsState> {
   }
 }
 
-function mapStateToProps(
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  state: Record<string, any>,
-  ownProps: INodePoolsProps
-) {
-  return {
-    nodePool: state.entities.nodePools.items[ownProps.nodePool.id],
```

Comment:
we were passing in an actual nodePool object and no reason to fetch it from state again?

Reply:
Yeah... not sure what the initial logic behind that was